semantic.cabal

```diff
@@ -25,20 +25,22 @@ flag release
 -- GHC extensions shared between targets
 common haskell
   default-language:    Haskell2010
-  default-extensions:  DataKinds
-                     , DeriveFoldable
-                     , DeriveFunctor
-                     , DeriveGeneric
-                     , DeriveTraversable
-                     , FlexibleContexts
-                     , FlexibleInstances
-                     , MonadFailDesugaring
-                     , MultiParamTypeClasses
-                     , OverloadedStrings
-                     , RecordWildCards
-                     , StandaloneDeriving
-                     , StrictData
-                     , TypeApplications
```

Comment:
All of these have been moved to the `.hs` files, except `StrictData`, which does not affect parsing/typechecking, and `MonadFailDesugaring`, which is obsolete in 8.6+.